scene_seg_model:
  ros__parameters:
    model_path: "../data/models/SceneSeg_FP32.onnx"
    backend: "tensorrt"
    precision: "fp16"
    model_type: "segmentation"
    input_topic: "/sensors/video/image_raw"
    output_topic: "/autoseg/scene_seg/mask"
    gpu_backend: "cuda"

domain_seg_model:
  ros__parameters:
    model_path: "../data/models/DomainSeg_FP32.onnx"
    backend: "tensorrt"
    precision: "fp16"
    model_type: "segmentation"
    input_topic: "/sensors/video/image_raw"
    output_topic: "/autoseg/domain_seg/mask"
    gpu_backend: "hip"